src/ui/components/pages/MyServices/MyServicesCards/MyServicesCard/MyServicesCard.tsx

```diff
@@ -40,6 +42,7 @@ export type Props = {
     ownerUsername: string | undefined;
     vaultTokenExpirationTime: number | undefined;
     s3TokenExpirationTime: number | undefined;
+    isUpgradable: boolean;
```

Comment:
I mean the condition that tells if the chart is upgradable can be deduced by looking if onRequestUpgrade() is `undefined` or not.  
`isUpgradable` is overdeterministic and introduce unecessary coupling ( !!isUpgradable => onRequestUpgrade !== undefined)